test/framework/setup.go

```diff
@@ -144,6 +144,24 @@ spec:
 	return YamlK8s(mesh)
 }
 
+func MTLSMeshKubernetesWithEgressRouting(name string) InstallFunc {
```

Comment:
Maybe we can start using `samples.MeshMTLSBuilder().WithEgressRoutingEnabled()` for the new codes instead of adding another type here

https://github.com/kumahq/kuma/issues/11294

Reply:
samples.MeshMTLSBuilder().WithEgressRoutingEnabled() returns a Resource now not a yaml... Let's do this as part of https://github.com/kumahq/kuma/issues/11294